src/app/catch/Health.js

```diff
@@ -84,14 +84,18 @@ define([
 
             this.inherited(arguments);
         },
-        getData: function () {
+        getData: function (controlMappings) {
```

Comment:
What's the purpose of adding this parameter if [you are setting `this.controlMappings` in `MoreInfoDialog`](https://github.com/agrc/electrofishing/pull/102/files#diff-34d17d815f577ae41232c39ad127d376R197)?

Reply:
I'm passing a control from the hard part collection tab into the health data flow. I define the controls in the moreinfodialog, because that is where it lives, and passing it into the health getdata method, because that is where the data is stored. health has it's own controlmapping so I pass the controls from the moreinfodialog to the health widget so they act as if they were on the same tab.

Reply:
I'm still not following but I'll take your word for it!

Reply:
let me show you